workflow/calculation/bb_sim.py

```diff
@@ -389,51 +392,57 @@ def unfinished():
         )
         lf_acc = np.copy(lf.acc(stat.name, dt=bb_dt))
         hf_acc = np.copy(hf.acc(stat.name, dt=bb_dt))
-        station_yaml = os.path.join(args.site_specific_dir, f"{stat.name}.yaml")
-        if os.path.isfile(station_yaml):
+        station_yaml = os.path.join(str(args.site_specific_dir), f"{stat.name}.yaml")
```

Comment:
Hmm, is this worthwhile having in the if statement with a walrus?

Well, someway of not creating it if site_specific_dir isn't True